notes/stories/film/film-scifi-2020s.st.txt

```diff
@@ -379,3 +379,48 @@ parental love [Richard pointedly proclaimed his love to his son Ethan.]
 choosing who lives and who dies [Regina Jackson selected the "chosen few" who would be sent 407 years into the future to reboot civilization.]
 
 
+movie: Breach (2020)
+====================
+
+:: Title
+Breach
+
+:: Description
+Breach, also titled Anti-Life, is an American science fiction action horror
+film about a group of 300,000 survivors of an extinction level event plague on
+Earth who embark on a voyage that will take them to a new colony called New
+Earth.
+
+:: Date
+2020-12-18
+
+:: References
+https://en.wikipedia.org/wiki/Breach_(2020_film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+natural pandemic [A main premise of the film is that 300,000 survivors of an extinction level event plague, which is running wild on Earth, are selected to be taken to a distant habitable planet called New Earth.]
+space colonization [A main premise of the film is that a group of 300,000 people leave a dying Earth on an interstellar ark in the hopes of colonizing a distant habitable planet called New Earth.]
+interstellar space travel [The story follows the crew of a massive interstellar ark, called the Ark, as they go on a six month journey to the distant planet New Earth.]
+boyfriend and girlfriend [Noah stowed away on the spaceship in large part to be with his pregnant girlfriend Hayley.]
+romantic love [Noah and and Hayley were two 20-somethings in love.]
+speculative spaceship [The story is set aboard a massive interstellar ark that was transporting 300,000 colonists to a habitable planet called New Earth.]
+extraterrestrial being [It turned out that the zombies were acting under the agency of a grotesque alien creature.]
+zombie [Everyone on the interstellar ark started turning into flesh eating, mindless zombies.]
+human vs. zombie [It was up to Clay, Noah and a handful of other crew members to fend off hoards of zombies that were running amok aboard the interstellar ark.]
+commander and subordinate [The no-nonsense Commander Stanley was pointedly strict with the crew members under his charge, especially Noah. Clay and his friends made moonshine behind the Commander Stanley's back.]
+
+:: Minor Themes
+expecting parents [While not much explored in the story, Noah had knocked up the admiral's daughter, Hayley, and they were expecting with child as a result.]
```

Comment:
Typo "expecting"

Reply:
I was expecting to find a typo in there, but I don't see.